pegasus/sites.v3/code.org/public/about/careers.md.erb

```diff
@@ -0,0 +1,143 @@
+    <div class="careers__values-tile careers__dark-gray">
+      <img class="careers__values-icon" src="../images/values/ValuesIcon6.png" alt=""/>
+      <p>We believe broad impact requires thinking big and acting holistically</p>
+    </div>
+  </div>
+</div>
+
+<h2 class="careers-header">Live Your Best Life</h2>
+<div class="careers__section-container" >
+  <div class="careers__benefits-container">
+    <div class="careers__benefits-tile-container">
+      <p class="careers__benefits-tile careers__benefits-left careers__teal">
+        Health Benefits for You and Your Family
+      </p>
+      <div class="careers__benefits-decorator careers__teal">
+        <img class="careers__benefits-icon" src="../images/careers-at-code/benefits-health.png" alt=""/>
+      </div>
+    </div>
+    <div class="careers__benefits-tile-container">
+      <div class="careers__benefits-decorator careers__green">
+        <img class="careers__benefits-icon" src="../images/careers-at-code/benefits-flex.png" alt=""/>
+      </div>
+      <p class="careers__benefits-tile careers__benefits-right careers__green">
+        Flexible, Supportive Work Environment
+      </p>
+    </div>
+    <div class="careers__benefits-tile-container">
+      <p class="careers__benefits-tile careers__benefits-left careers__orange">
+        Generous PTO and Leave Plans
+      </p>
+      <div class="careers__benefits-decorator careers__orange">
+        <img class="careers__benefits-icon" src="../images/careers-at-code/benefits-pto.png" alt=""/>
+      </div>
+    </div>
+    <div class="careers__benefits-tile-container">
+      <div class="careers__benefits-decorator careers__purple">
+        <img class="careers__benefits-icon" src="../images/careers-at-code/benefits-pd.png" alt=""/>
+      </div>
+      <p class="careers__benefits-tile careers__benefits-right careers__purple">
+        Growth and Learning
+      </p>
+    </div>
+  </div>
+
+  <p>
+    We care about our employees and are dedicated to supporting everyone both at work and outside of work. We know that everyone has different needs, which is why we offer a holistic benefits package that will ensure that you (and your family) are taken care of.
+    </p>
+</div>
+
+
+<h2 class="careers-header" id="roles">Come Work with Us</h2>
+
+Code.org is based in Seattle, but many of our team members are remote; remote candidates are welcome and encouraged to apply!
+
+<div id='lever-jobs-container'></div>
+<script type='text/javascript'>
+
+  window.leverJobsOptions = {accountName: 'Code.org'};
+
+</script>
+<script type='text/javascript' src='https://andreasmb.github.io/lever-jobs-embed/index.js'></script>
+
+<div class="careers__future">
+  We're always on the lookout for people who want to help make our vision a reality! Even if we don't have a job posted right now that matches your skills, we'd still love to hear from you via <a href="https://jobs.lever.co/code.org/6ca7d2c4-9965-4cf0-960e-de58e3d0d829">this application</a>.
```

Comment:
Should this be a `<p>` element?

Reply:
(same question for a few others above and below)

Reply:
I would think "yes", but it appears that our markdown renderer that's applied to this content will auto-wrap these in a `<p>` element. Being explicit within the code doesn't appear to impact it in any way, but does appear to be unnecessary. I think this is part of the unique-ness of this being both a markdown and an erb template. I'm happy to do whatever the "standard" is, though!